### PR TITLE
Adding meeting minutes motion

### PR DESCRIPTION
**Pull request by sstevens2**
for the Dec 2022 special election meeting and the Q1 2023 Meeting

@kariljordan can you review this?  Also I missed doing a PR for the other motion I just added (and its subsequent corrections), can you check that it looks okay?

### CODE REVIEW

File: minutes/motions.md [hunk 1]
```diff
@@ -3,6 +3,18 @@ The following file includes all motions passed during Executive Council meetings
 
 ## 2023 Motions
 
+### Approval Dec 2022 Special Election Meeting Minutes and of the Q1 2023 Meeting Minutes
+
+This is a motion to approve Dec 2022 Special Election Meeting Minutes and of the Q1 2023 Meeting Minutes.
+
+**I move to approve the  Dec 2022 Special Election Meeting Minutes and of the Q1 2023 Meeting Minutes.**
```
Comment (sstevens2) [inline minutes/motions.md:10]:
```suggestion
**I move to approve the Dec 2022 Special Election Meeting Minutes and the Q1 2023 Meeting Minutes.**
```

File: minutes/motions.md [hunk 2]
```diff
@@ -3,6 +3,18 @@ The following file includes all motions passed during Executive Council meetings
 
 ## 2023 Motions
 
+### Approval Dec 2022 Special Election Meeting Minutes and of the Q1 2023 Meeting Minutes
+
+This is a motion to approve Dec 2022 Special Election Meeting Minutes and of the Q1 2023 Meeting Minutes.
```
Comment (sstevens2) [inline minutes/motions.md:8]:
```suggestion
This is a motion to approve Dec 2022 Special Election Meeting Minutes and the Q1 2023 Meeting Minutes.
```

File: minutes/motions.md [hunk 3]
```diff
@@ -3,6 +3,18 @@ The following file includes all motions passed during Executive Council meetings
 
 ## 2023 Motions
 
+### Approval Dec 2022 Special Election Meeting Minutes and of the Q1 2023 Meeting Minutes
```
Comment (sstevens2) [inline minutes/motions.md:6]:
```suggestion
### Approval of the Dec 2022 Special Election Meeting Minutes and the Q1 2023 Meeting Minutes
```

**Review by kariljordan — APPROVED** [review]
Thank you Sarah!

**Pull request merged**